CHANGELOG.md

```diff
@@ -2,12 +2,12 @@
 
 ### v1.4.5 Apr 25, 2024
 
-* Upgrade Reader SDK version support to 1.7.2 for iOS
+* Upgrade Reader SDK version support to 1.7.4 for iOS
```

Comment:
This changelog doesn't seem right. I don't think we updated iOS to 1.7.4 in April of 2024 and Android to 1.7.7 in Aug of 2023, because we are updating in this PR.  I think you should undo these changes in the changelog and add a new entry with the correct plugin version and date it will be pushed